source/grid/grid_generator.cc

```diff
@@ -5360,25 +5361,96 @@ namespace GridGenerator
 
     std::map<typename MeshType<dim - 1, spacedim>::cell_iterator,
              typename MeshType<dim, spacedim>::face_iterator>
-      surface_to_volume_mapping;
+      surface_to_volume_mapping; // return of the function
```

Comment:
Since this is not modified until the end I would just move the declaration down to that point.

Reply:
Agreed